src/Command/ValidateCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Command;
+
+use GraphQL\Error\InvariantViolation;
+use Overblog\GraphQLBundle\Resolver\TypeResolver;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ValidateCommand extends Command
+{
+    use RequestExecutorLazyLoaderTrait;
+
+    private $typeResolver;
+
+    public function __construct(TypeResolver $typeResolver)
+    {
+        parent::__construct();
+        $this->typeResolver = $typeResolver;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('graphql:validate')
+            ->setDescription('Validate schema')
+            ->addOption(
+                'schema',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The schema name to validate.'
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $schemaName = $input->getOption('schema');
+        $schema = $this->getRequestExecutor()->getSchema($schemaName);
+
+        try {
+            @$schema->assertValid();
```

Comment:
Can you comment why are you using `@` here please?

Reply:
I rework this one to use a cleaner and more explicit method :
Warning::suppress(true);